### DISCUSSION TITLE
API Creazione fattura Campi addizionali XML

### DISCUSSION THREAD

**Discussion by imbundle** [fattureincloud/api#458] [Q&A] [answered]
Buon pomeriggio,
se volessi creare una fattura tramite api e volessi ad esempio aggiungere il campo
2.1.2(1).2 - IdDocumento:
è corretto usare la voce
ei_raw
se si come dovrebbe essere popolato questo campo in modo da soddisfare la richiesta del cliente di inserire quel campo nella fattura inviata?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @imbundle 
sì, è corretto.

Il valore dovrebbe essere il seguente:

```json
"ei_raw": {
  "FatturaElettronicaBody": {
    "DatiGenerali": {
      "DatiOrdineAcquisto": [
        {
          "IdDocumento": "VALUE"
        }
      ]
    }
  }
}
```

Può trovare maggiori informazioni qui: [E-Invoice XML customization](https://developers.fattureincloud.it/docs/guides/e-invoice-xml-customisation)

Grazie
Mauro
